packages/block-editor/src/components/image-size-control/index.js

```diff
@@ -30,6 +31,11 @@ export default function ImageSizeControl( {
 	onChange,
 	onChangeImage = noop,
 } ) {
+	deprecated( 'wp.blockEditor.__experimentalImageSizeControl', {
+		since: '6.3',
+		alternative:
+			'wp.blockEditor.privateApis.DimensionsTool and wp.blockEditor.privateApis.ResolutionTool',
```

Comment:
I just noticed this message when testing the "Media & Text" block, which still uses the deprecated component.

This `alternative` suggestion isn't helpful for third-party consumers since the private APIs aren't available for them. Should we remove this from the deprecation message?

Reply:
Also just noting here since I just ran into this yesterday, it isn't fun for third party extenders when a component is deprecated whilst the replacement is still marked as private 😞